babe

```diff
@@ -236,7 +253,8 @@ def scale(appName, scale, force=False):
         printApiError(r)
 
 def killInstance(appName, instance, grace, rangerOOR, lbOOR, scale, fabric=False):
-    endpoint=adminEndpoint(instance)
+    isNodeInstance = isNodeApp(appName)
+    endpoint = (adminEndpoint(instance) if not isNodeInstance else instanceEndpoint(instance))
```

Comment:
adminEndpoint should itself abstract out this detail instead of handling this in multiple places.

Reply:
Fix all places that use this

Reply:
Moved the logic into the function and renamed adminEndpoint to oorEndpoint